frontend/src/components/data-table/column-header.tsx

```diff
@@ -33,11 +33,8 @@ export const DataTableColumnHeader = <TData, TValue>({
     return <div className={cn(className)}>{header}</div>;
   }
 
-  const sortFn = column.getSortingFn();
-  const AscIcon =
-    sortFn === sortingFns.basic ? ArrowDown01Icon : ArrowDownNarrowWideIcon;
-  const DescIcon =
-    sortFn === sortingFns.basic ? ArrowDown10Icon : ArrowDownWideNarrowIcon;
```

Comment:
not sure if it was a regression from BE sorting. or another change, but this logic doesnt work well inferring the icon to show, so instead just always default to `ArrowDownNarrowWideIcon/vice-versa`